src/page-content/navigation/navigation-header.tsx

```diff
@@ -32,198 +34,211 @@ export const sideNav: Pick<NavProps, 'sideNav'> = {
       style={'filled'}
       openInNewWindow={false}
     >
-      Request Access
+      Sign Up
     </Button>
   ]
 }
 
 export const dropdownMenu: Pick<NavProps, 'dropdownMenu'> = {
   dropdownMenu: [
     {
-      title: 'Product',
+      title: 'Ory Cloud',
       mainMenu: [
         {
           iconLeft: Cloud,
-          title: 'Ory Cloud',
+          title: 'Sign Up',
           description: 'Never Build Auth Again',
           to: 'https://console.ory.sh/registration'
         },
+        {
+          iconLeft: Pricing,
+          title: 'Pricing',
+          description: "Ory Cloud Pricing",
+          to: '/pricing'
+        }
+      ]
+    },
+    {
+      title: 'Ory Open Source',
```

Comment:
```suggestion
      title: 'Open Source',
```
just call it open source?